maven-core/src/main/java/org/apache/maven/plugin/PluginParameterExpressionEvaluator.java

```diff
@@ -244,6 +246,10 @@ else if ( "mojoExecution".equals( expression ) )
         {
             value = mojoExecution;
         }
+        else if ( "artifactFilter".equals( expression ) )
```

Comment:
We should investigate if we can solve this with MPLUGIN-302 instead of exposing this a new expression.